src/output.cpp

```diff
@@ -46,13 +68,20 @@ void writeProgramInfo(std::ostream& out) {
 /*
     Used to update the board on screen
 */
-void refreshBoard(const BoardType& board,
+void refreshBoard(const BoardType& playerBoard,
+                  const BoardType& computerBoard,
+                  const BoardType& currentBoard,
                   const char* const boardDiscription,
                   OutputTranslationTable translationTable) {
+    if (&currentBoard != &playerBoard || &currentBoard != &computerBoard) {
```

Comment:
The part is throwing invalid arguments on game start @jotoho

Reply:
@TimBeckmann You're using the wrong logic operator. We need `&&`, not `||`

Reply:
oh